docs/modules/operation/pages/bsm/business-service-hierarchy.adoc

```diff
@@ -1,15 +1,15 @@
 
-= Business Service Hierarchy
+= Business service hierarchy
 
-BS can depend on each other and together build a business service hierarchy.
-Visualize it using the Topology User Interface with the Business Services View.
-The operational status of a BS is ultimately calculated from alarms and their serverity.
-To define the class of alarms a reduction key is used and is represented as an edge of a BS.
+Business services can depend on each other and together build a business service hierarchy.
+Visualize the hierarchy using the Topology User Interface with the business services view.
```

Comment:
I just did a search and replace on "BS" to change it to business services, but didn't reference the UI. Thanks for point this out. Since this is a menu option, then the style should be Bold. I'll update.